templates/circa/templates.yaml

```diff
@@ -0,0 +1,55 @@
+dataset: circa
+templates:
+  053260a8-1bcc-4805-81d2-bb528fc56ca2: !Template
+    id: 053260a8-1bcc-4805-81d2-bb528fc56ca2
+    jinja: 'Convert the question to declarative form:
+
+
+      {{question_X}} |||
+
+      {{canquestion_X}}'
+    name: question_declarative
+    reference: ''
+  70b7a94a-6a39-4a81-9a6e-0709a0acdb28: !Template
+    id: 70b7a94a-6a39-4a81-9a6e-0709a0acdb28
+    jinja: "What is the most common judgement in a less strict way given the question-answer\
+      \ pair of X and Y in the context of {{context}}?\n\nX: {{question_X}} \nY: {{answer_Y}}\
+      \ |||\n\n{{   \n [\n  \"Yes\",\n  \"No\",\n  \"In the middle, neither yes nor\
+      \ no\",\n  \"Yes, subject to some conditions\",\n  \"Other\"\n]\n [goldstandard2]\
+      \ }}"
+    name: goldstandard2_judgement
+    reference: ''
+  73466d0f-b1b1-4c61-8f03-346e121ae06c: !Template
+    id: 73466d0f-b1b1-4c61-8f03-346e121ae06c
+    jinja: 'What is a possible question to ask someone given the context of {{context}}?
+      |||
+
+      {{question_X}}'
+    name: possible_qn
+    reference: ''
+  997f7f96-d420-48c1-85f7-ecade54adbd7: !Template
+    id: 997f7f96-d420-48c1-85f7-ecade54adbd7
+    jinja: "What is the most common judgement given the question-answer pair of X\
+      \ and Y in the context of {{context}} ?\n\nX: {{question_X}} \nY: {{answer_Y}}\
+      \ |||\n\n{{goldstandard1}}"
```

Comment:
You need to map `goldstandard1` to a string

Reply:
Sorry I dont get this. Please help to clarify
goldstandard1 is the index for judgements in the list: [
  "Yes",
  "No",
  "In the middle, neither yes nor no",
  "Probably yes / sometimes yes",
  "Probably no",
  "Yes, subject to some conditions",
  "Other",
  "I am not sure how X will interpret Y’s answer"
]

Reply:
Right, but here you are asking the model to generate the index rather than the string

Reply:
It seems to generate the string itself such as "Yes" and "No". You can take a look at the Prompted dataset viewer.
![image](https://user-images.githubusercontent.com/43485111/121060057-b9516900-c7f4-11eb-8ef1-44e2e59716b6.png)

Reply:
sorry I got confused between goldstandard1 and this
Corrected now!